sys/include/net/ng_ndp/types.h

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_ng_ndp_types    Types for IPv6 neighbor discovery
+ * @ingroup     net_ng_ndp
+ * @brief       IPv6 neighbor discovery message types
+ * @{
+ *
+ * @file
+ * @brief       IPv6 neighbor discovery message type definitions
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef NG_NDP_TYPES_H_
+#define NG_NDP_TYPES_H_
+
+#include <inttypes.h>
```

Comment:
Usually, I'm a fan of putting the includes into the C files where they are needed and avoid includes in header files as much as possible. But in the end it's a matter of taste.